backend/apps/content/views.py

```
@@ -536,3 +546,36 @@ class QuizDraftViewSet(viewsets.ModelViewSet):
     queryset = QuizDraft.objects.all()
     serializer_class = QuizDraftSerializer
     permission_classes = [permissions.AllowAny]
+
+
+class LearningPathViewSet(viewsets.ModelViewSet):
+    serializer_class = LearningPathSerializer
+    permission_classes = [permissions.IsAuthenticatedOrReadOnly]
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)